sub-packages/bionemo-esm2/tests/bionemo/esm2/model/test_model.py

```diff
@@ -39,9 +38,6 @@
 from bionemo.testing import megatron_parallel_state_utils
 
 
-nemo1_checkpoint_path: Path = load("esm2/nv_650m:1.0")
```

Comment:
Would it make sense to make this a fixture? I'm ok either way.

Reply:
I think it's better to just put these load() calls in where they're used. That way we only download the data if/when we run the tests 🤷